tests/parm/control_csawmg.nml.IN

```diff
@@ -0,0 +1,303 @@
+﻿&amip_interp_nml
```

Comment:
this namelist is not used. can be removed from all input.nml namelist templates

Reply:
To my knowledge, this nml is unused.

Reply:
Rusty, thanks for the explanation. The amip_interp_nml is removed in the updated global tests.

Reply:
Shouldn't `amip_interp_nml` be removed from all input_nml templates, not just global?